ffi/diplomat/src/bidi.rs

```diff
@@ -60,6 +61,14 @@ pub mod ffi {
                 Level::new(default_level).ok(),
             )))
         }
+        /// Utility function for producing reorderings given a list of levels
+        ///
+        /// Produces a map saying which visual index maps to which source index.
+        #[diplomat::rust_link(unicode_bidi::Level::is_rtl, FnInStruct)]
+        pub fn reorder_visual(&self, levels: &[u8]) -> Box<ICU4XReorderedIndexMap> {
+            let levels = Level::from_slice_unchecked(levels);
```

Comment:
Question: is it safe to use a method named "unchecked" here in this context where you definitely haven't performed any checking on the &[u8] ?